src/Schema/Tables/Sessions.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE TABLE [dbo].[Sessions]
+(
+	[UserId] INT NOT NULL PRIMARY KEY, 
+    [Token] VARCHAR(50) NOT NULL, 
```

Comment:
NVARCHAR as we have spoken today, more preferable.

Reply:
For token VARCHAR is ok. We do not expect Cyrillic symbols here.